firmware/lucidgloves-firmware/_main.ino

```diff
@@ -42,6 +42,31 @@ void getInputs(void* parameter){
 }
 #endif
 
+#if (ENABLE_PCA_9865_SERVO && USING_FORCE_FEEDBACK)
+  #pragma once
+  #include "advancedConfig.h"
```

Comment:
I don't like this portion here, but could not get the firmware to compile without it.

Reply:
That is pretty wacky, also it should be `AdvancedConfig`... you sure you need to #include it there?

Reply:
Your right, I just commented it out and compiled and it's unneeded. No idea why I needed it before/what I was doing different in April